R/convenience-functions.R

```diff
@@ -0,0 +1,48 @@
+#' @title Add transformations
+#'
+#' @description Add transformations of the forecasts and observations for
+#' later scoring. For more information on why this might be desirable, check
+#' out the linked reference.
+#'
+#' @inheritParams score
+#' @param fun A function used to transform both true values and predictions
```

Comment:
I think a lot of the value for this comes from adding a few default options along with justification. The aim of the package is to be helpful right - at the moment this is such thin wrapper it isn't clear it adds much.